{"frames":[{"func":"ggerrorplot(dpl3, x = month, y = per.var2020_otros, desc_stat = \"mean_sd\")","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"do.call(.plotter, .opts)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"(function (fun, data, x, y, combine = FALSE, merge = FALSE, color = \"black\", \n    fill = \"white\", title = NULL, xlab = NULL, ylab = NULL, legend = NULL, \n    legend.title = NULL, facet.by = NULL, select = NULL, remove = NULL, \n    order = NULL, add = \"none\", add.params = list(), label = NULL,  ...","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":".check_data(data, x, y, combine = combine | merge != \"none\")","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"stop(\"Can't find the y elements in the data.\")","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error in .check_data(data, x, y, combine = combine | merge != \"none\") : \n  Can't find the y elements in the data.\n"}